experiments/rsc/src/app/AdminPage.tsx

```diff
@@ -1,3 +1,25 @@
-export default function AdminPage(): React.ReactNode {
-    return 'admin page'
+import CreateUser from "./components/CreateUser";
+import Login from "./components/Login";
+import { db } from "../db";
+
+export default async function AdminPage() {
+    const users = await db.selectFrom('User').select(['id', 'name', 'cellnumber']).execute();
+    const isAuthenticated = true;
```

Comment:
Ah, ok so we stub this for now?

Reply:
Ja I just put it there for some "context"